packages/apps/graph-client/src/constants/non-fungible.ts

```diff
@@ -0,0 +1 @@
+export const NONFUNGIBLETRANSACTION: string = 'marmalade-v2.ledger';
```

Comment:
Let's underscore constants:

```suggestion
export const NON_FUNGIBLE_TRANSACTION: string = 'marmalade-v2.ledger';
```